server/package.json

```diff
@@ -55,8 +55,5 @@
     "underscore": "1.8.3",
     "valid-url": "1.0.9",
     "winston": "1.0.2"
-  },
-  "optionalDependencies": {
-    "polisServerBrand": "git+https://github.com/colinmegill/polisServerBrand.git#master"
```

Comment:
Thanks! I see that you removed this dep as well. Might you be willing to remove the usage of the package named `optional` from codebase, as that's what loaded `polisServerBrand`. Then this could close https://github.com/pol-is/polisServer/issues/25 as well? :)

Reply:
sure, just pushed that into this pr

I'm not super good at package-lock.json - should I run npm install to update it? The trouble is I've been coding on my laptop but deploying on my server and so I'm not sure the changes to package-lock will be desirable

I guess I'll try to just update the part that removes the deps

Reply:
nvm it was pretty straightforward when i ran on laptop; done